react/features/device-selection/DeviceSelectionPopup.js

```diff
@@ -164,10 +166,13 @@ export default class DeviceSelectionPopup {
      * Returns Promise that resolves with true if the device change is available
      * and with false if not.
      *
+     * @param {string} [deviceType] - Values - 'output', 'input' or undefined.
+     * Default - 'input'.
```

Comment:
I'd be in favor of rewording this doc a bit because I thought _default 'input'_ meant by default the string "input" would be passed in to sendRequest but really what's returned by default is device change availability for input devices.